alacritty/src/config/color.rs

```diff
@@ -140,7 +140,7 @@ impl Default for FocusedMatchColors {
     fn default() -> Self {
         Self {
             background: CellRgb::Rgb(Rgb { r: 0x00, g: 0x00, b: 0x00 }),
-            foreground: CellRgb::Rgb(Rgb { r: 0xff, g: 0xff, b: 0xff }),
+            foreground: CellRgb::Rgb(Rgb { r: 0xf4, g: 0xbf, b: 0x75 }),
         }
     }
```

Comment:
This is not what you have in the configuration file and I'd assume the config values would be much more distinguishable.